packages/moonstone/Input/Input.module.less

```diff
@@ -69,8 +73,6 @@
 	}
 
 	&.small {
-		padding: 0 (@moon-input-small-height / 2);
```

Comment:
Small should probably have its own padding rather than inheriting using large's padding. What do you think?

Reply:
I don't think there's specs for padding differences between normal and small. I think the normal padding still works for small, unless you have another padding in mind.

Reply:
Nah, I'm ok with that. I love consistency. :)